docs/install/installing-istio.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "Installing Istio for Knative"
+weight: 15
+type: "docs"
+---
+
+This guide walks you through manually installing and customizing Istio for use
+with Knative.
+
+If your cloud platform offers a managed Istio installation, we recommend
+installing Istio that way, unless you need the ability to customize your
+installation. If your cloud platform offers a managed Istio installation,
+the [install guide](./README.md) for your specific platform will have those instructions.
+
+For example, the [GKE Install Guide](./knative-with-gke.md) includes the
+instructions for installing Istio on your cluster using `gcloud`.
+
+## Before you begin
+
+You need:
+- A Kubernetes cluster created.
+- [`helm`](https://helm.sh/) installed.
+
+## Installing Istio
+
+When you install Istio, there are a few options depending on your goals. For a
+basic Istio installation suitable for most Knative use cases, follow the
+[Installing Istio without sidecar injection](#installing-istio-without-sidecar-injection)
+instructions. If you're familiar with Istio and know what kind of
+installation you want, read through the options and choose the installation that
+suits your needs.
+
+You can easily customize your Istio installation with `helm`. The below sections
+cover a few useful Istio configurations and their benefits. 
+
+### Choosing an Istio installation
+
+You can install Istio with or without a service mesh:
+
+- _automatic sidecar injection_: Enables the Istio service mesh by
+  [automatically injecting the Istio sidecars][1]. The sidecars are injected
+  into each pod of your cluster as they are created.
+
+- _manual sidecar injection_: Provides your Knative installation with traffic
+  routing and ingress, without the Istio service mesh. You do have the option of
+  later enabling the service mesh if you [manually inject the Istio sidecars][2].
+
+If you are just getting started with Knative, you should choose automatic
```

Comment:
This statement seems conflict with https://github.com/knative/docs/pull/1314/files#diff-1116bcf887cf6b5e21e5e4f96fa026b1R28 because installing Istio withouth sidecar injection means no automatic sidecar injection.

Reply:
Whoops, good catch! I switched it back and forth too many times :)